mdk/README.txt

```diff
@@ -16,14 +16,14 @@ Step 1: Open your command-line and browse to the folder where you extracted the
 
 Step 2: You're left with a choice.
 If you prefer to use Eclipse:
-1. Run the following command: `gradlew genEclipseRuns` (`./gradlew genEclipseRuns` if you are on Mac/Linux)
 2. Open Eclipse, Import > Existing Gradle Project > Select Folder 
+1. Run the following command: `./gradlew genEclipseRuns`
    or run `gradlew eclipse` to generate the project.
```

Comment:
Does `gradlew eclipse` not work anymore? Or well, I guess it doesn't in FG6. I don't remember what its deal is in FG5.

Reply:
Dunno, I don't use neither Eclipse or FG5 anymore :P